sr_controllers_tools/src/sr_controllers_tools/sr_controller_helper.py

```diff
@@ -137,3 +148,19 @@ def change_force_ctrl_type(self, chng_type_msg):
             rospy.logwarn("Failed to change the control type.")
 
         return success
+
+    def change_arm_teach_mode(self, teach_mode):
+        success = True
+#         set_teach_msg = SetTeachMode()
```

Comment:
please remove commented out code

Reply:
Removed in f90032a18513273eacc759a02a5878f7ca7b91d5.